lib/internal/event_target.js

```diff
@@ -96,13 +96,11 @@ class Event {
    *   composed?: boolean,
    * }} [options]
    */
-  constructor(type, options = null) {
+  constructor(type, options = {}) {
```

Comment:
Would this be a candidate for `kEmptyObject` from internal/util.js?

Reply:
Yes, I think so. Thanks for the swift feedback, updated! :-)